docs/yaml/elementary/str.yml

```diff
@@ -41,6 +41,35 @@ methods:
     description: The values to replace the @number@ placeholders in the format string.
     type: int | bool | str
 
+# str.hash(hash_algo)
+- name: hash
+  description: Run the given `hash_algo` on the string and return the hash constant
```

Comment:
"constant"? Also, most (all?) ashing algorithms consume and return bytes. Here it seems that an hexadecimal representation of the hash is returned instead.